src/main/kotlin/org/rust/ide/annotator/fixes/AddMutableFix.kt

```diff
@@ -0,0 +1,41 @@
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.RsNamedElement
+import org.rust.lang.core.psi.ext.parentOfType
+
+open class AddMutableFix(val binding: RsNamedElement) : LocalQuickFixAndIntentionActionOnPsiElement(binding) {
+    override fun getFamilyName(): String = text
+    override fun getText(): String  = "Add mutable"
+    open val mutable = true
+
+    override fun invoke(project: Project, file: PsiFile, editor: Editor?, startElement: PsiElement, endElement: PsiElement) {
+        updateMutable(project, binding, mutable)
+    }
+}
+
+fun updateMutable(project: Project, binding: RsNamedElement, mutable: Boolean = true) {
+    when (binding) {
+        is RsPatBinding -> {
+            val parameter = binding.parentOfType<RsValueParameter>()
```

Comment:
Judging by the test it should work correctly. Let's perhaps add a couple of tests with complex patterns in functions, to make sure we don't do something funny? 

Possible tests would be

```Rust
fn foo((x, y): (i32, i32)) {
    x = 92;
}

fn foo(&mut (ref x, ): &mut (String, )) {
    x.push_str("x");
}
```

Probably there are other interesting ones!